### PR TITLE
Data reviews in human readable form

### PR DESCRIPTION
**Pull request by alvesitalo**
### Pull Request checklist ###

- [x] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry to `HISTORY.rst` or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

Fix: #243

### CODE REVIEW

File: glean_parser/markdown.py [hunk 1]
```diff
@@ -121,6 +122,28 @@ def ping_data_reviews(
         return None
 
 
+def ping_review_title(data_url: str, index: int) -> str:
+    """
+    Return a title for a data review in a readable form.
```
Comment (mdboom) [inline glean_parser/markdown.py:127]: Can you also document the parameters passed into this function?

**Review by mdboom — CHANGES_REQUESTED** [review]
Just one minor comment improvement, otherwise this looks great!

File: glean_parser/markdown.py (hunk 1 above)
Comment (alvesitalo) [reply glean_parser/markdown.py]: Done! Thanks

**Review by mdboom — APPROVED** [review]

**Review by mdboom — CHANGES_REQUESTED** [review]
It looks like it's caught some linting issues now: https://app.circleci.com/pipelines/github/mozilla/glean_parser/1035/workflows/ca73e673-0c0f-416d-b21c-edf8d02db8ac/jobs/3014

**Comment by alvesitalo** [comment]
All tests passed now 🎉

**Review by mdboom — APPROVED** [review]

**Pull request merged**